bundles/org.openhab.binding.webexteams/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.webexteams-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-webexteams" description="WebexTeams Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<feature>openhab-transport-http</feature>
```

Comment:
I am not sure this is required. Why did you add that line ?

Reply:
I saw that in the Spotify binding and somehow associated it with the use of the servlet.  I've removed it and it seems to work just fine.